pkg/kev/log/log.go

```diff
@@ -258,12 +235,10 @@ func decorate(level string, f ...Fields) logrus.Fields {
 		switch level {
 		case "debug-title":
 			fields["prefix"] = successTitlePrefix
-		case "debug-detail":
+		case "debug":
 			fields["prefix"] = detailPrefix
```

Comment:
As `DebugPrefix` is now removed perhaps it makes sense to address/change other prefixes too, i.e. drop the emojis and replace with something else to be consistent?

Reply:
of course.

Reply:
IMHO, that should be work in a separate PR.

E.g. The way error logging works in the `converter` (and some places in `render`) doesn't entirely conform to the new UX.

We have a UX ticket on the backlog which that can address the above while revamping logging for our text user interface.

Reply:
@marcinc ^